src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss

```diff
@@ -6,51 +6,52 @@
 @mixin __checkboxFacade($pseudo-state: null) {
   @if $pseudo-state == focus {
     border-color: $blue-900;
-    border-color: var(--hxCheckbox-bordercolor-focus, $blue-900);
+    border-color: $rax-color-primary-900;
     box-shadow: $focus-glow;
   }
   @else if $pseudo-state == hover {
-    background-color: var(--hxCheckbox-hover-bgcolor, $gray-0);
-    border-color: var(--hxCheckbox-hover-bordercolor, $blue-700);
-    color: var(--hxCheckbox-hover-color, $blue-700);
+    background-color: $rax-component-checkbox-hover-background-color;
+    border-color: $rax-component-checkbox-hover-border-color;
+    color: $blue-700;
+    color: $rax-component-checkbox-hover-border-color;
   }
 }
 
 // ========== INVALID ==========
 @mixin __checkboxFacade--invalid($pseudo-state: null) {
   @if $pseudo-state == focus {
     border-color: $red-status-500;
-    border-color: var(--hxCheckbox-invalid-bordercolor, $red-status-500);
+    border-color: $rax-component-checkbox-invalid-border-color;
     box-shadow: $focus-glow-invalid;
   }
   @else if $pseudo-state == hover {
     border-color: $red-status-500;
-    border-color: var(--hxCheckbox-invalid-hover-bordercolor, $red-status-500);
+    border-color: $rax-component-checkbox-invalid-border-color;
     color: $red-status-500;
-    color: var(--hxCheckbox-invalid-hover-color, $red-status-500);
+    color: $rax-component-checkbox-invalid-border-color;
   }
   @else {
     border: 1px solid $red-status-500;
-    border: 1px solid var(--hxCheckbox-invalid-border, $red-status-500);
+    border: 1px solid $rax-component-checkbox-invalid-border-color;
     color: $red-status-500;
-    color: var(--hxCheckbox-invalid-color, $red-status-500);
+    color: $rax-component-checkbox-invalid-border-color;
   }
 }
 
 // ========== DISABLED ==========
 @mixin __checkboxFacade--disabled($pseudo-state: null) {
   @if $pseudo-state == focus {
     border-color: $gray-500;
-    border-color: var(--hxCheckbox-disabled-bordercolor, $gray-500);
+    border-color: $rax-component-checkbox-disabled-border-color;
```

Comment:
Please continue to use the IE11 fallback pattern. See above.

Reply:
Changes have been done, please review.